<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QWidget" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinPosKeeper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="sliderAutoCapturePeriod">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAutoCapturePeriod">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCaptureNow">
       <property name="text">
        <string>&amp;Capture now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestore">
       <property name="text">
        <string>&amp;Restore state</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearLogs">
       <property name="text">
        <string>Clear &amp;logs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditLogs">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>logs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="windowsWidgets">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClearLogs</sender>
   <signal>clicked()</signal>
   <receiver>textEditLogs</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
